src/js/components/Page.js

```diff
@@ -28,14 +31,28 @@ var Page = React.createClass({
       rendered: true
     });
     this.forceUpdate();
+    SidebarStore.addChangeListener(
```

Comment:
this seems like something we've already abstracted with the `StoreMixin`. let's just add the `SidebarStore` to the `StoreConfig` and use it from there